tests/src/testfeelement.hh

```diff
@@ -135,10 +127,14 @@ inline auto checkJacobianFunctor = [](auto& nonLinOp, [[maybe_unused]] auto& fe,
   auto subOperator = nonLinOp.template subOperator<1, 2>();
   return checkJacobianOfElement(subOperator);
 };
-inline auto checkLinearStressFunctor
-    = [](auto& nonLinOp, auto& fe, [[maybe_unused]] auto& req) { return checkLinearStress(nonLinOp, fe); };
-
 inline auto checkResultFunctionFunctor
     = [](auto& nonLinOp, auto& fe, [[maybe_unused]] auto& req) { return checkResultFunction(nonLinOp, fe); };
 inline auto checkFEByAutoDiffFunctor
     = [](auto& nonLinOp, auto& fe, auto& req) { return checkFEByAutoDiff(nonLinOp, fe, req); };
+
+auto checkCalculateAtFunctorFactory(const auto resultCollectionFunction) {
+  return [=](auto& nonLinOp, auto& fe, [[maybe_unused]] auto& req) {
```

Comment:
The way this is constructed here, I think it is sufficient to capture by reference `&` isn't it?

Reply:
It actually is.